build/windows/installer/conf/fluent-bit.conf

```diff
@@ -45,5 +45,18 @@
     Match oms.container.log.la.*
     Exclude stream ${AZMON_LOG_EXCLUSION_REGEX_PATTERN}
 
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
+#${KubernetesFilterEnabled}    Merge_Log           Off
+#${KubernetesFilterEnabled}    Buffer_Size         0
```

Comment:
Did you validate that, this works for windows containers? if not, lets validate this.

Reply:
yes A value of 0 results in no limit, and the buffer will expand as-needed. For windows container, the data is flowing good